addons/project/views/project_sharing_views.xml

```diff
@@ -89,7 +89,7 @@
                                 <div t-if="record.date_deadline.raw_value" name="date_deadline" attrs="{'invisible': [('is_closed', '=', True)]}">
                                     <field name="date_deadline" widget="remaining_days"/>
                                 </div>
-                                <div t-if="record.displayed_image_id.value">
+                                <div t-if="record.displayed_image_id.value" groups="base.group_user">
```

Comment:
Could you update the commit message accordingly:
```text
[IMP] project: restrict the visibility of `displayed_image_id` in project_sharing

This commit prevent showing the `displayed_image_id` to non internal users.

task-2895909
```

Reply:
> Could you update the commit message accordingly:
> 
> ```
> [IMP] project: restrict the visibility of `displayed_image_id` in project_sharing
> 
> This commit prevent showing the `displayed_image_id` to non internal users.
> 
> task-2895909
> ```

Done